README.md

```diff
@@ -1,2 +1,100 @@
 # Rest7Contract
-Repository to discuss the new REST API contract for DSpace 7 https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group
+Repository to discuss the new REST API contract for DSpace 7 <https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group>
+
+## Use of the HTTP Verbs and HTTP Response CODE
+
+### On collection of resources endpoints
+- POST
+Adds a new element to the collection.
+
+- GET
+Returns the first page of the resources in the collection
+
+### On single resource endpoints
+- GET
+Returns a single entity.
+
+- HEAD
```

Comment:
Is this checking for existence or is this a mechanism to check for authorization?  An example would be useful.